File: motor3d 2.0/objetoescena.vb
Namespace Escena
    Public MustInherit Class ObjetoEscena
        Inherits Global.Motor3D.ObjetoBase

        Public Shadows Event Modificado(ByRef Sender As ObjetoEscena)

        Public Sub New()
            MyBase.New()
        End Sub

        Public Overrides Function ToString() As String
            Return MyBase.ToString()
        End Function
    End Class
End Namespace

File: motor3d 2.0.ConvertedToC#/ObjetoEscena.cs
using Microsoft.VisualBasic;
using System;
using System.Collections;
using System.Collections.Generic;
using System.Data;
using System.Diagnostics;
using System.Linq;
using System.Xml.Linq;
namespace Motor3D.Escena
{
	public abstract class ObjetoEscena : global::Motor3D.ObjetoBase
	{

		public new event ModificadoEventHandler Modificado;
		public new delegate void ModificadoEventHandler(ref ObjetoEscena Sender);

		public ObjetoEscena() : base()
		{
		}

		public override string ToString()
		{
			return base.ToString();
		}
	}
}

